src/rules/at-use-no-unnamespaced/index.js

```diff
@@ -0,0 +1,29 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("at-use-no-unnamespaced");
+
+export const messages = utils.ruleMessages(ruleName, {
+  expected: "Expected @use to use a namespace"
```

Comment:
I went and checked from the stylelint repo, and most of the `-no-` rules seem to use the `rejected: "Unexpected ..."` message format, should this rule also use it?

Reply:
I updated it